lessons/basics/modules.md

```diff
@@ -6,13 +6,15 @@ order: 8
 lang: en
 ---
 
-We know from experience it's unruly to have all of our functions in the same file and scope.  In this lesson we're going to cover how to group functions and define a specialized map known as a struct in order to organize our code more efficiently.
+We know from experience it's unruly to have all of our functions in the same file and scope.
```

Comment:
Why are we putting every sentence on a new line?  This creates a huge amount of unnecessary changes on each PR.  If there isn't a good reason for this let's stop.